### PR TITLE
[14.0][ADD] mail_server_by_user

### PR DESCRIPTION
**Pull request by ChrisOForgeFlow**
Added module to change smtp server depends on user, is system find smtp server equals to email that user performing actions, changes mail_server
@ForgeFlow

### CODE REVIEW

**Review by olgamarcocb — APPROVED** [review]
LGTM :+1:

File: mail_server_by_user/views/mail_server_view.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (MiquelRForgeFlow) [inline mail_server_by_user/views/mail_server_view.xml]: delete this empty file, right?

**Comment by rafaelbn** [comment]
Hello @ChrisOForgeFlow , comments from @MiquelRForgeFlow are pending thank  you!

**Comment by ChrisOForgeFlow** [comment]
> Hello @ChrisOForgeFlow , comments from @MiquelRForgeFlow are pending thank you!

Done!, thanks @MiquelRForgeFlow

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]

**Comment by rafaelbn** [comment]
@ChrisOForgeFlow @MiquelRForgeFlow could you check please the fail here: https://github.com/OCA/social/pull/828#issuecomment-1065976122 ? Thank you!

**Comment by CasVissers-360ERP** [comment]
Any new plans to merge this PR?

**Comment by MiquelRForgeFlow** [comment]
@rafaelbn can you reopen this PR, please?

**Comment by rafaelbn** [comment]
Done @MiquelRForgeFlow !

**Comment by MiquelRForgeFlow** [comment]
Thanks. Then we can proceed to merge this PR before another undesired closing, right? hehe

**Pull request merged**